debian/vanilla-installer.install

```diff
@@ -1,3 +1,4 @@
 recipe.json etc/vanilla-installer/
 installer.log etc/vanilla/
-abroot-adapter usr/bin/
\ No newline at end of file
+abroot-adapter usr/bin/
```

Comment:
I think this can be removed since abroot-adapter was removed

Reply:
Oops